core/src/main/java/org/polypheny/db/util/Util.java

```diff
@@ -1819,7 +1819,7 @@ public static void debugCode( PrintStream out, String code ) {
                 out.print( "/*" );
                 String number = Integer.toString( i );
                 if ( number.length() < 4 ) {
-                    Spaces.append( out, 4 - number.length() );
+                    out.append( StringUtils.rightPad( "", 4 - number.length() ) );
                 }
                 out.print( number );
                 out.print( " */ " );
```

Comment:
The lines from `out.print( "/*" );` to `out.print( " */ " );` can be replaced by
```java
out.printf( "/*%-4d */", i );
```
unless I am missing something.  The import for `StringUtils` can then be dropped as well.